app/routes/report-routes.ts

```diff
@@ -95,9 +96,12 @@ const extractTestResult = (req: Request): TestResult | undefined => {
 };
 
 const createReportRateLimit = rateLimit({
-  max: 20, // allowed requests per window
-  windowMs: 24 * 60 * 60 * 1000, // 24 hour window,
-  keyGenerator: req => determineRemoteAddress(req)
+  max: config.RATE_LIMIT_COUNT, // allowed requests per window
+  windowMs: config.RATE_LIMIT_WINDOW, // window length in miliseconds
+  keyGenerator: req => determineRemoteAddress(req),
+  onLimitReached(req, res) {
+    return res.redirect(`${res.locals.urls.limit}`);
```

Comment:
Nice, you don't really need the template literals here, `return res.redirect(res.locals.urls.limit);` will do

Reply:
nice catch, thank you, fixing